test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -313,6 +313,136 @@ function setup-addon-manifests {
   chmod 644 "${dst_dir}"/*
 }
 
+# Write the config for the audit policy.
+# Note: This duplicates the function in cluster/gce/gci/configure-helper.sh.
+# TODO: Reuse the original one instead.
```

Comment:
This seems pretty ugly. Can you file an issue for unifying this with the original and link it here?

Reply:
Filed https://github.com/kubernetes/kubernetes/issues/53321.
An earlier discussion about this - https://github.com/kubernetes/kubernetes/pull/52998#issuecomment-331940632